kazoo_couch/priv/couchdb/views/account-crossbar_listings.json

```
@@ -643,9 +643,12 @@
                 "",
                 "      /* attributes/mailbox_number */",
                 "      /* vmboxes/listing_by_mailbox */",
-                "      additionalEmitList.push({",
-                "        'key': [doc.pvt_type, doc.mailbox]",
-                "      });",
+                "      var aliases = doc.aliases || [];",
+                "      for (var element in aliases.push(doc.mailbox)) {",
```

`Array.prototype.push` returns a new length value, not a new array, so this doesn't seem like it will work correctly. I don't see any `"vmbox"` entries in the view

---

good catch, he probably meant to write: 

```suggestion
                "      for (var element in aliases.concat(doc.mailbox)) {",
```